definitions/checks/foreman/check_checkpoint_segments.rb

```diff
@@ -0,0 +1,50 @@
+module Checks
+  module Foreman
+    class CheckpointSegments < ForemanMaintain::Check
+      metadata do
+        label :check_checkpoint_segments
+        description 'Check if checkpoint_segments parameter persent in custom hiera'
+        tags :pre_upgrade
```

Comment:
Can we limit this check based on postgresql service using confine block?

Reply:
Yes, that can be done. 
I will update the PR.

Reply:
Yes